### DISCUSSION TITLE
Connessione con lampade Bluetooth

### DISCUSSION THREAD

**Discussion by davix10** [indomus/forum#2176] [HUB: Home Assistant]
Dal terminal di HA trovo il mio dispositivo bluetooth (lampadina) ma come faccio ad integrarlo e/o a fare il pairing? Non mi compare nulla in dispositivi e servizi

![image](https://user-images.githubusercontent.com/6436570/207585319-a30f2d75-adf6-48ae-aa31-330ca11b5f72.png)

**Comment by 50494554524F** [comment]
ovviamente devono essere modelli supportati https://www.home-assistant.io/integrations/#light

se non lo sono cerca un'integrazione in hacs e se non trovi niente ti resta da cercare su github o smanettare con il protocollo se è pubblico

**Reply by davix10** [reply]
> ovviamente devono essere modelli supportati https://www.home-assistant.io/integrations/#light
> 
> se non lo sono cerca un'integrazione in hacs e se non trovi niente ti resta da cercare su github o smanettare con il protocollo se è pubblico

Se trovassi il protocollo di comunicazione come dovrei fare per connettermi? Esiste una guida per capire come iniziare a comunicare?

Grazie
